### PR TITLE
sorter: rename some structures and methods to make revive happy

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve? 

None

### What is changed and how it works?

trivial refactor for unified sorter.

rename some structures and methods to make happy.

### Check List 

Tests 

 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change

Side effects

None

Related changes

None

### Release note 

```release-note

None

```

### CODE REVIEW

**Comment by hi-rustin** [comment]
/cc @ben1009  

/label require-LGT1

Could you please take a look? Thanks!

**Review by ben1009 — APPROVED** [review]

**Comment by hi-rustin** [comment]
/assign @amyangfei 

Could you please help me trigger the merge process? Thanks!

**Review by amyangfei — APPROVED** [review]

**Review by lonng — APPROVED** [review]

**Comment by hi-rustin** [comment]
/run-kafka-integration-test

signal: segmentation fault (core dumped)

File: cdc/sorter/unified/unified_sorter.go [hunk 1]
```diff
@@ -34,9 +34,8 @@ const (
 	heapCollectChSize = 128 // this should be not be too small, to guarantee IO concurrency
 )
 
-// UnifiedSorter provides both sorting in memory and in file. Memory pressure is used to determine which one to use.
-//nolint:revive
-type UnifiedSorter struct {
+// Sorter provides both sorting in memory and in file. Memory pressure is used to determine which one to use.
+type Sorter struct {
```
Comment (liuzix) [inline cdc/sorter/unified/unified_sorter.go:38]: `Sorter` might be too generic a name and might hinder readability?

Comment (hi-rustin) [reply cdc/sorter/unified/unified_sorter.go:38]: It's easy to distinguish, because our pkg is `unified`.

unified.Sorter
memory.Sorter

**Pull request merged**